modules.d/90kernel-modules/module-setup.sh

```diff
@@ -19,13 +20,19 @@ installkernel() {
         IFS=$_OLDIFS
     }
 
-    is_block_dev() {
-        [ -e /sys/dev/block/$1 ] && return 0
+    record_block_dev_drv() {
+        for _mod in $(get_dev_module /dev/block/$1); do
+            [[ " $_hostonly_drvs " != *$_mod* ]] && _hostonly_drvs+=" $_mod"
+        done
+        [[ "$_hostonly_drve" ]] && return 0
```

Comment:
typo! `_hostonly_drve` vs `_hostonly_drvs`

Reply:
Thanks! I found this issue when testing but forgot to patch it up... Sorry, my bad.